diff --color -Naur sonic-screenlocker.old/CMakeLists.txt sonic-screenlocker/CMakeLists.txt
--- sonic-screenlocker.old/CMakeLists.txt	2026-05-23 02:41:08.346994414 +0000
+++ sonic-screenlocker/CMakeLists.txt	2026-05-23 02:43:15.704255243 +0000
@@ -2,10 +2,10 @@
 
 project(kscreenlocker)
 set(CMAKE_C_STANDARD 99)
-set(PROJECT_VERSION "6.6.5")
+set(PROJECT_VERSION "6.6.4")
 set(PROJECT_VERSION_MAJOR 6)
 
-set(PROJECT_DEP_VERSION "6.6.5")
+set(PROJECT_DEP_VERSION "6.6.4")
 set(QT_MIN_VERSION "6.10.0")
 set(KF6_MIN_VERSION "6.22.0")
 
